### ISSUE TITLE
💅 `biome-ignore` supresses other instances of a lint rule violation

### ISSUE THREAD

**Issue by jennings** [biomejs/biome#5608]
### Environment information

```bash
Playground
```

### Rule name

Any, used lint/suspicious/noConsole

### Playground link

https://next.biomejs.dev/playground/?lintRules=all&code=YwBvAG4AcwBvAGwAZQAuAGwAbwBnACgAIgBzAGgAbwB1AGwAZAAgAGUAcgByAG8AcgAiACkAOwAKAAoALwAvACAAYgBpAG8AbQBlAC0AaQBnAG4AbwByAGUAIABsAGkAbgB0AC8AcwB1AHMAcABpAGMAaQBvAHUAcwAvAG4AbwBDAG8AbgBzAG8AbABlADoAIABmAG8AbwAKAGMAbwBuAHMAbwBsAGUALgBsAG8AZwAoACIAcwBoAG8AdQBsAGQAIABiAGUAIABzAHUAcABwAHIAZQBzAHMAZQBkACIAKQA7AA%3D%3D

### Expected result

In this playground, there are two violations of noConsole being suppressed. The single `// biome-ignore` suppresses both of them, but should only suppress the first one.

If you remove the comment, both violations appear.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by ematipico** [comment]
Duplicate of https://github.com/biomejs/biome/issues/5562
